server/settings/tools.py

```diff
@@ -162,7 +162,7 @@ class AYONMenuModel(BaseSettingsModel):
     version_up_current_workfile: bool = SettingsField(
         False,
         title="Version Up Workfile",
-        description="Add 'Version Up Workfile' to AYON menu"
+        description="Adds tool for 'incremental save' to the AYON menu"
```

Comment:
It's not really a "tool" because it doesn't really open any dialog?

```suggestion
        description="Adds menu item for 'incremental save' to the AYON menu"
```